themes/root/templates/Captcha/figlet.phtml

```diff
@@ -0,0 +1,7 @@
+<?php
+  $laminasCaptcha = $this->captcha->getCaptcha();
+  $id = $laminasCaptcha->generate();
+?>
+<pre><?=$laminasCaptcha->getFiglet()->render($laminasCaptcha->getWord())?></pre>
+<p>Please enter what you see: <input name="<?=$this->captcha->getHtmlInputId()?>"></p>
```

Comment:
We need to add a translation string and `transEsc()` call for the "Please enter what you see:" label.

Reply:
We should also use a `<label>` tag to associate the label with the input.